Source/WebInspectorUI/UserInterface/Views/LayoutTimelineDataGridNode.js

```diff
@@ -60,6 +60,19 @@ WI.LayoutTimelineDataGridNode = class LayoutTimelineDataGridNode extends WI.Time
         switch (columnIdentifier) {
         case "name":
             cell.classList.add(...this.iconClassNames());
+
+            if (this.record.eventType == WI.LayoutTimelineRecord.EventType.LargestContentfulPaint) {
+                let fragment = document.createDocumentFragment();
+                fragment.append(value);
+
+                if (this.record.domNode) {
+                    let goToButton = WI.createGoToArrowButton();
+                    fragment.appendChild(goToButton);
+                    goToButton.addEventListener("click", this._goToArrowWasClicked.bind(this), false);
```

Comment:
the `false` is not necessary

also, rather than use `bind` we could just use an arrow function
```js
goToArrow.addEventListener("click", (event) => {
    WI.showMainFrameDOMTree(domNode, {ignoreSearchTab: true});
});
```